source/uwp/AdaptiveCardTestApp/Views/TestResultView.xaml.cs

```diff
@@ -60,8 +77,30 @@ private void TestResultView_DataContextChanged(FrameworkElement sender, DataCont
 
         private async void ButtonSaveAsExpected_Click(object sender, RoutedEventArgs e)
         {
+            bool updateOriginals = false;
+            bool updateImage = false;
+            bool updateJson = false;
+
+            Button buttonSender = sender as Button;
+            if (buttonSender.Name == "ButtonUpdateOriginals")
```

Comment:
>if (buttonSender.Name == "ButtonUpdateOriginals") [](start = 12, length = 49)

Instead of doing string comparison, you should be able to do the following... compare the reference

```
if (sender == ButtonUpdateOriginals)
```

That way it'll also be compiled and checked in case we ever rename the buttons :) The following could also work if the reference comparison provides a warning

```
if (buttonSender.Name == ButtonUpdateOriginals.Name)
```